Tools/Generator/BidiTypeLookupGenerator.cpp

```diff
@@ -70,7 +70,7 @@ const string BidiTypeLookupGenerator::BranchDataSegment::hintLine() const {
     return ("/* INDEX_BLOCK: -- 0x" + Converter::toHex(index, 4) + "..0x" + Converter::toHex(index + dataset->size() - 1, 4) + " -- */");
 }
 
-BidiTypeLookupGenerator::BidiTypeLookupGenerator(const Parser::DerivedBidiClass &derivedBidiClass)
+BidiTypeLookupGenerator::BidiTypeLookupGenerator(const DerivedBidiClass &derivedBidiClass)
```

Comment:
This change may look surprising but it was previously incorrect (in this file we have `using namespace SheenBidi::Parser` but no `using namespace SheenBidi`) and started to fail likely because we're now compiling in a more standards-compliant mode.